voice-change-o-matic/LICENSE

```diff
@@ -0,0 +1,116 @@
+CC0 1.0 Universal
```

Comment:
I believe this is the same license that the repository as a whole uses so, we can remove it here.

Reply:
good 👍🏻

Reply:
done in https://github.com/mdn/webaudio-examples/pull/92/commits/053ab323bd89e51dfecaac43e8157dcec784b00a